.stylelintrc

```diff
@@ -16,8 +17,10 @@
         ]
       }
     ],
+    "color-function-notation": null,
     "custom-property-empty-line-before": "never",
     "no-eol-whitespace": null,
+    "selector-class-pattern": null,
```

Comment:
Stylelint now doesn't like `.fooBar` selectors (it wants `.foo-bar`).

I figured we know what we're doing and don't need this rule. But if we want it we could supply a different regex to match whatever pattern we want.